tracer/samples/NugetDeployment/HttpListenerExample.csproj

```diff
@@ -8,7 +8,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Datadog.Monitoring.Distribution" Version="1.28.2" />
+    <PackageReference Include="Datadog.Trace.Bundle" Version="2.16.0" />
```

Comment:
Should we keep this up to date with the version bump target?

Reply:
I asked myself the question. I don't see the added value honestly as we don't compile or run them. But I'm ok either way honestly